js/preloader.js

```diff
@@ -1,147 +1,178 @@
-    
-    goScrollToBottom();
+  count++;
+  
+  if (repeat == 0) {
+    if (count > 3) {
+      delay = 300;
+    }
     
-    count++;
+    if (count > 6) {
+      delay = 100;
+    }
     
-    if (repeat == 0) {
-      if (count > 3) {
-        delay = 300;
-      }
-      
-      if (count > 6) {
-        delay = 100;
-      }
-      
-      if (count > 8) {
-        delay = 50;
-      }
-      
-      if (count > 10) {
-        delay = 10;
-      }
-    } else {
-      if (count > 3) {
-        delay = 10;
-      }
+    if (count > 8) {
+      delay = 50;
     }
     
-    if (count < strings.length) {
-      setTimeout(function() {
-        return addLog();
-      }, delay);
-    } else {
-      setTimeout(function() {
-        delay = 1000;
-        return createLog("ok");
-      }, 1000);
+    if (count > 10) {
+      delay = 10;
+    }
+  } else {
+    if (count > 3) {
+      delay = 10;
     }
   }
   
-  function createLog(type, index) {
-    var row = document.createElement('div');
-    
-    var spanStatus = document.createElement('span');
-    spanStatus.className = type;
-    spanStatus.innerHTML = type.toUpperCase();
-    
-    var message = (index != null) 
-                ? strings[index] 
-                : 'kernel: Initializing...';
-
-    if(index == null) 
-    {
-      var preloader = $('#preloader');
-      jQuery(preloader).fadeOut("slow");
-      jQuery("#main").fadeIn("slow");
-    }
-    
-    var spanMessage = document.createElement('span');
-    spanMessage.innerHTML = message;
-    
-    row.appendChild(spanStatus);
-    row.appendChild(spanMessage);
-    
-    return row;
+  if (count < strings.length) {
+    setTimeout(function() {
+      return addLog();
+    }, delay);
+  } else {
+    setTimeout(function() {
+      delay = 1000;
+      return createLog("ok");
+    }, 1000);
   }
+}
+
+function createLog(type, index) {
+  var row = document.createElement('div');
+  
+  var spanStatus = document.createElement('span');
+  spanStatus.className = type;
+  spanStatus.innerHTML = type.toUpperCase();
   
-  function goScrollToBottom() {
-    $(document).scrollTop($(document).height()); 
+  var message = (index != null) 
+              ? strings[index] 
+              : 'kernel: Initializing...';
+
+  if(index == null) 
+  {
+    var preloader = $('#preloader');
+    jQuery(preloader).fadeOut("slow");
+    jQuery("#main").fadeIn("slow");
   }
   
-  addLog();
+  var spanMessage = document.createElement('span');
+  spanMessage.innerHTML = message;
+  
+  row.appendChild(spanStatus);
+  row.appendChild(spanMessage);
   
-  
\ No newline at end of file
+  return row;
+}
+
+function goScrollToBottom() {
+  $(document).scrollTop($(document).height()); 
+}
+
+function setCookie(cname,cvalue,exdays) {
+  var d = new Date();
+  d.setTime(d.getTime() + (exdays*24*60*60*1000));
+  var expires = "expires=" + d.toGMTString();
+  document.cookie = cname + "=" + cvalue + ";" + expires + ";path=/";
+}
+function getCookie(cname) {
```

Comment:
Please change the getCookie method to something more shorter. Probably something like [this](https://stackoverflow.com/a/25490531/6303750)
Also, do not forget to mention this link in the comment if you use this code.

Reply:
@ashawe thanks for the feedback and the reference I have made changes and mentioned the comment as well . please review